WalletWasabi.Documentation/WasabiCompatibility.md

```diff
@@ -10,14 +10,23 @@ This document lists all the officially supported software and devices by Wasabi
 - Fedora 33+
 - Debian 10+
 
-# Officially Supported Hardware wallets
+# Officially Supported Hardware wallets in Wasabi 1.0
 
 - ColdCard MK1
 - ColdCard MK2
 - ColdCard MK3
```

Comment:
MK4?

Reply:
That's a good question. I asked @adamPetho to figure out if we have any.

Reply:
We don't have an MK4 in the office, yet.

Reply:
@yahiheb confirmed MK4 works, so @karozagorus add it to the list!